### DISCUSSION TITLE
Your Team encountered an unknown problem. Please reach out to our support team for details. Learn More

### DISCUSSION THREAD

**Discussion by alikaz3mi** [vercel/community#6602] [Account Issue]
### Question

Hello. I'm getting this issue for my account. Could you examine it? 

I have read and sent message in `https://github.com/orgs/vercel/discussions/6063` too.

**Comment by RamiroPagella** [comment]
im having the same issue and i have sent a message in that discussion as well. My vercel id is: T8IUAGKI934MxJ1cuNFX5si7
